src/hooks/index.ts

```diff
@@ -33,9 +36,16 @@ import {generateShelleyPlateFromKey} from '../yoroi-wallets/cardano/shelley/plat
 import {YoroiSignedTx, YoroiUnsignedTx} from '../yoroi-wallets/types'
 
 // WALLET
-export const useCloseWallet = (options?: UseMutationOptions<void, Error>) => {
+export const useCloseWallet = ({onSuccess, ...options}: UseMutationOptions<void, Error> = {}) => {
+  const dispatch = useDispatch()
+
   const mutation = useMutation({
     mutationFn: () => walletManager.closeWallet(),
+    onSuccess: (data, variables, context) => {
+      dispatch(clearUTXOs())
```

Comment:
dont add dispatch to these hooks

Reply:
applied here -> https://github.com/Emurgo/yoroi-mobile/pull/2125